CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # Changelog
 ## Unreleased
 ## 1.6.1 (2021-02-19)
+### Misc updates
+- Amplify/Tools pod is currently deprecated and it will be completely removed in a future release, a deprecation warning will be issued if used in an Xcode build script
```

Comment:
```suggestion
- Amplify/Tools pod is currently deprecated and it will be completely removed in a future release. For now, Xcode will emit a deprecation warning if your project invokes AmplifyTools in a custom build phase.
```